apps/users/models/user/profile.py

```diff
@@ -36,7 +36,8 @@ class Profile(models.Model):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.states = ["subscribed_newsletter", "newsletter_recipient"]
-        self.update_states()
+        # TODO fix states
+        # self.update_states()
```

Comment:
I had some unfinished work on unit tests, which included starting to fix this. I pushed it to the feature/unit-test branch so you can take a look at it, but not sure what the current state is. It also contains some fixed up unit tests for the async_requests and some for users. If you prefer, I can also take a look at it later, but that might still be a while.

Reply:
I plan to create tests for the update views and models in a separate PR

Reply:
fair enough